### DISCUSSION TITLE
When I use morphine through the treatment interface, the morphine in the headset or PUCS is used first.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#13888] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [x] My issue happened while using mods.

### What happened?

This is the wrong order, obviously the morphine in the headset is an inexorable supply of last chance. It should be used last. The morphine in the inventory should be used first, then the one in the belt or medic uniform, and only lastly the one in the headset.
This error also appears with PUCS. If there is a medication in the PUCS injector (a bag of blood or morphine), it is used first when I using the treatment interface.

### Reproduction steps

1. Take morphine to inventory
2. Place morphine  to the headset's injector
3. Lets jump off a high ladder to get a wound, for which the interface will offer morphine.
4. Open treatment interface by pressing H
5. Click on morphine icon in treatment interface. Now you are cured.
6. Look in headset's injector. It is empty now.

The same sequence can be repeated with PUCS.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.4.5.0 (Blood in the Water Update, hotfix 1)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! I unfortunately could not reproduce this. This [was reported a while back and we implemented a fix for it](https://github.com/FakeFishGames/Barotrauma/issues/12951), and in my tests, it seemed to be working correctly - items in your inventory are preferred over the items in the headset or PUCS:

https://github.com/FakeFishGames/Barotrauma/assets/14351070/74b652d0-88c1-4bf3-a68d-dcd4db20478e

**Reply by pavel4win** [reply]
Thanks for the tip, I'll keep the medic uniform in my inventory instead of my belt.

I think the problem is that I keep my supplies in my medic uniform, and I keep my medic uniform in my tool belt.
Try doing this, I think in this case the error will appear.
It makes sense for the headset to be the last one, even if the medic's uniform (which contains medications) is in the tool belt.
If you follow the balance of the game, you should prohibit placing clothes (with pockets) on the tool belt. After all, now you can store 6x3x8 = 144 medications of 18 different types on your belt.
This is a clear imbalance. And let them throw stones at me.
To be honest, clothes should generally be prohibited from being kept in inventory; clothes should only be held in both hands.
Add more pockets to clothes and prohibit clothes from being kept in inventory. There is even such a mod.
https://steamcommunity.com/sharedfiles/filedetails/?id=2500953748

Pay attention to the improved syringe gun, which fits on a tool belt and can contain 16 syringes of two types. Total 16x6 = 96
Should it be allowed to hold this on a tool belt?

**Comment by pavel4win** [comment]
I realized that the medical interface does not see at all those medications that are in medical clothing placed in the tool belt. Is'nt it?

**Comment by pavel4win** [comment]
After removing the mod "BetterHealthUI (Updated)"
https://steamcommunity.com/sharedfiles/filedetails/?id=3231293294
I checked again the sequence of using medications from the medical interface.
Everything works in the right order.
First, medications are used from the inventory, then from the belt, and finally from the headset.
The BetterHealthUI (Updated) mod was to blame for the incorrect order. The discussion can be closed.
I apologize for creating extra work.
